<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudgetTracker</class>
 <widget class="QWidget" name="BudgetTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>888</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>733</width>
    <height>888</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>888</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BudgetTracker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="transactionHLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="transactionGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Table</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="transactionTableView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>440</width>
              <height>0</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="transactionVLayout">
         <item>
          <widget class="QGroupBox" name="entryGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>239</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Add Entry</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="entryVLayout">
              <item>
               <layout class="QHBoxLayout" name="entryDateHLayout">
                <item>
                 <widget class="QLabel" name="entryDateLabel">
                  <property name="text">
                   <string>Date</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="entryDateDateEdit">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>125</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="currentSection">
                   <enum>QDateTimeEdit::YearSection</enum>
                  </property>
                  <property name="displayFormat">
                   <string>yyyy/MM/dd</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="entryCategoryHLayout">
                <item>
                 <widget class="QLabel" name="entryCategoryLabel">
                  <property name="text">
                   <string>Category</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="entryCategoryLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>125</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="entrySubcategoryHLayout">
                <item>
                 <widget class="QLabel" name="entrySubcategoryLabel">
                  <property name="text">
                   <string>Subcategory</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="entrySubcategoryLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>125</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="entryAmountHLayout">
                <item>
                 <widget class="QLabel" name="entryAmountLabel">
                  <property name="text">
                   <string>Amount</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="entryAmountLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>125</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="entryButtonHLayout">
                <item>
                 <widget class="QPushButton" name="entryAddButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="entryRemoveButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="tableFilterGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>239</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Filter Table</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="tableFilterVLayout">
              <item>
               <layout class="QHBoxLayout" name="tableFilterCategoryHLayout">
                <item>
                 <widget class="QLabel" name="tableFilterCategoryLabel">
                  <property name="text">
                   <string>Category</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="tableFilterCategoryLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>125</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="tableFilterSubcategoryHLayout">
                <item>
                 <widget class="QLabel" name="tableFilterSubcategoryLabel">
                  <property name="text">
                   <string>Subcategory</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="tableFilterSubcategoryLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>125</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="tableFilterButtonHLayout">
                <item>
                 <widget class="QPushButton" name="tableFilterFilterButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Filter</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="tableFilterClearButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Clear Filter</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="plotFilterGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>239</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Filter Plot</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QVBoxLayout" name="plotFilterVLayout_inner">
              <item>
               <layout class="QHBoxLayout" name="plotFilterCategoryHLayout">
                <item>
                 <widget class="QLabel" name="plotFilterCategoryLabel">
                  <property name="text">
                   <string>Category</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="plotFilterCategoryLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>125</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="plotFilterSubcategoryHLayout">
                <item>
                 <widget class="QLabel" name="plotFilterSubcategoryLabel">
                  <property name="text">
                   <string>Subcategory</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="plotFilterSubcategoryLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>125</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>125</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maxLength">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="plotFilterButtonHLayout">
                <item>
                 <widget class="QPushButton" name="plotFilterFilterButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Filter</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="plotFilterClearButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Clear Filter</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="plotHLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="plotGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Plot</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCustomPlot" name="transactionPlot" native="true">
            <property name="minimumSize">
             <size>
              <width>440</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
